P5/Source/Guidelines/en/FS-FeatureStructures.xml

```diff
@@ -1176,15 +1205,14 @@ using an XML identifier are discussed in <ptr target="#SAUR"/></note>
 way of accomplishing this is to add an XML identifier to each
 <gi>fsDecl</gi> element in <ident type="file">example.xml</ident>:
 <egXML xmlns="http://www.tei-c.org/ns/Examples" valid="feasible" source="#UND">
-           <!-- ... -->
-           <fsdDecl>
-           <fsDecl type="gpsg" xml:id="GPSG">
+<fsdDecl>
```

Comment:
Ah, I just 'fixed' (or so I thought) it in passing, because it looked so bad in the chapter that I thought I should do something about it :-/
The was some bad indentation, and I just tried to make the result better. I can happily withdraw that.

Reply:
Oh, so far, my impression was that you usually want to conserve space and let the formatter do some automatic work, except for the "feasible" cases, where it doesn't. Maybe I misinterpreted. Not sure what to do here. I shouldn't have touched it. Let me please leave it spoiled, and return to this example and others at some later point. 
Lesson learned: stay within your scope.  :-\